drivers/fastboot/fb_mmc.c

```diff
@@ -29,6 +29,27 @@ static int part_get_info_by_name_or_alias(struct blk_desc *dev_desc,
 {
 	int ret;
 
+
+	/** check if the MMC is configured for boot partition r/w.
+	 *  If so, then we aren't reading the partition table, it's not
+	 *  stored in the boot partition only the bootloader is and
+	 *  instead we are in raw block programming mode of the HW
+	 *  bootpartition, most likely to program the bootloader. If we
+	 *  try to read the partition from emmc while the emmc HW boot
+	 *  partition is activit, it'll just fail. Not a big deal, but
+	 *  the failure is a timeout in SDHC, so it takes a couple
+	 *  seconds to fail. Checking for that here prevents the query
+	 *  and saves the 2 seconds failure.
+	 *  See eMMCC spec for partition configuration registers (CSD
+	 *  Byte 179)
+	 */
+	struct mmc *mmc = find_mmc_device(CONFIG_FASTBOOT_FLASH_MMC_DEV);
+	if ( mmc && (EXT_CSD_EXTRACT_PARTITION_ACCESS(mmc->part_config))){
```

Comment:
Hi Pat,

Sorry to be picky but the coding style here looks bad and doesn't match uboot/kernel standard.

You can use  scripts/checkpatch.pl to check the coding style. Right now this patch produces:
total: 3 errors, 16 warnings, 1 checks, 165 lines checked

Thanks